### PR TITLE
Fix storage file datalake readme and samples issues

### PR DESCRIPTION
**Pull request by v-xuto**
Fix issue https://github.com/Azure/azure-sdk-for-python/issues/11358
1. Add the correct link for "Azure Portal", "Azure PowerShell", '"Azure CLI". All three correspond to the same link 'Azure Portal'. 
Update links about "Azure PowerShell" and '"Azure CLI": 
Azure PowerShell: https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-powershell, 
Azure CLI: https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-cli
2. Modify the CLI command for creating the storage account.

### CODE REVIEW

**Comment by v-xuto** [comment]
@jongio for notification.

File: sdk/storage/azure-storage-file-datalake/README.md [hunk 1]
```diff
@@ -25,17 +25,20 @@ pip install azure-storage-file-datalake --pre
...
+[Azure PowerShell](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-powershell),
+or [Azure CLI](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-cli):
 
 ```bash
 # Create a new resource group to hold the storage account -
 # if using an existing resource group, skip this step
 az group create --name my-resource-group --location westus2
 
+# Install the extension “Storage-Preview”
+az extension add --name storage-preview
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-file-datalake/README.md:38]: what is this used for?

File: sdk/storage/azure-storage-file-datalake/README.md [hunk 2]
```diff
@@ -25,17 +25,20 @@ pip install azure-storage-file-datalake --pre
 
 ### Create a storage account
 If you wish to create a new storage account, you can use the
-[Azure Portal](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-the-azure-portal),
-[Azure PowerShell](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-powershell),
-or [Azure CLI](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-azure-cli):
+[Azure Portal](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal),
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-file-datalake/README.md:28]: why do we need to change the link? The existing link is under "How to" under ADLS Gen2, after it's changed it's under blob storage.

File: sdk/storage/azure-storage-file-datalake/README.md (hunk 1 above)
Comment (v-xuto) [reply sdk/storage/azure-storage-file-datalake/README.md:38]: According to this, [MicrosoftDocs/azure-docs#24481](https://github.com/MicrosoftDocs/azure-docs/issues/24481), CLI extension is required.

File: sdk/storage/azure-storage-file-datalake/README.md (hunk 2 above)
Comment (v-xuto) [reply sdk/storage/azure-storage-file-datalake/README.md:28]: These two links point to the same page. You can open it and have a look.

Comment (xiafu-msft) [reply sdk/storage/azure-storage-file-datalake/README.md:28]: but the tabs are different right?

Comment (v-xuto) [reply sdk/storage/azure-storage-file-datalake/README.md]: I have updated "Azure Portal" link.

Comment (xiafu-msft) [reply sdk/storage/azure-storage-file-datalake/README.md]: Could you please revert the changes for powershell and azure CLI as well? Or is there's a reason you want to change those two?

File: sdk/storage/azure-storage-file-datalake/README.md (hunk 1 above)
Comment (xiafu-msft) [reply sdk/storage/azure-storage-file-datalake/README.md:38]: I added a comment on that issue, it looks like MicrosoftDoc needs to be updated.

File: sdk/storage/azure-storage-file-datalake/README.md (hunk 2 above)
Comment (v-xuto) [reply sdk/storage/azure-storage-file-datalake/README.md]: About Powershell and azure CLI , after the two links are opened, the corresponding pages are all about the steps of azure portal. You can have a try. The reason why I changed it is to refer to the SDK in other languages, like [net sdk storage file datalake readme](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Files.DataLake#prerequisites).

**Comment by v-xuto** [comment]
@xiafu-msft - What are your thoughts on this PR?

**Review by xiafu-msft — APPROVED** [review]

**Pull request merged**